src/core/connectors.js

```diff
@@ -1181,5 +1181,17 @@ define(function() {
 		label: 'KEXP Radio',
 		matches: ['*://kexp.org', '*://www.kexp.org', '*://kexp.org/*', '*://www.kexp.org/*'],
 		js: ['connectors/kexp.js'],
+	}, {
+		label: 'Radio Rocks',
+		matches: [
```

Comment:
Would `www.radioroks.ua/player/*` be enough? For me I am always seeing www redirection and would cover further channels.

Reply:
I try, no effect